postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresDBRecord.java

```diff
@@ -91,9 +114,17 @@ protected void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldIn
       stmt.setObject(sqlIndex, createPGobject(columnType.getTypeName(),
                                               record.get(field.getName()),
                                               stmt.getClass().getClassLoader()));
-    } else {
-      super.writeToDB(stmt, field, fieldIndex);
+      return;
+    } else if (columnType.getType() == Types.NUMERIC) {
+      if (record.get(field.getName()) != null) {
```

Comment:
nit: more readable to combine these all into a single clause to reduce the amount of nesting

Reply:
Done